R/subset_datarqst.R

```diff
@@ -15,9 +15,17 @@
 #' @param Species Character vector of six letter species codes. Include all species that are requested.
 #' @param filter Character vector of unique population species combinations (in the format PopID_Species).
 #' Include all unique population species combinations requested.
-#'
-#' @return Create 4 .csv files at the location of the standard data. Files will be given a unique number
-#' based on the date when the subset was performed.
+#' @param include_conflicting is FALSE. If include_conflicting = TRUE, individuals of conflicting species which
+#' were identified as a species contained in `Species` at least once are included in the subset of data. If
+#' include_conflicting = FALSE, these individuals are removed.
+#' @param output_type is 'R' and can be set to 'csv'. If output_type is 'csv' 4 .csv files will be created in the save path.
+#' If output_type is 'R' an .RDS file will be created in the save path, and an R object in the
+#' running R session if return_R = TRUE.
+#' @param return_R is FALSE. If return R = TRUE, the subset of data is also saved to the current
+#' R Session as an R object (list).
+#' @param save is TRUE. If save = FALSE, the subset of data is not saved into the save path.
```

Comment:
Do we need all these save options? The idea of the function is to create a subset of the standard format so I would think that we would *always* want to save to path. Thoughts?

Reply:
@LiamDBailey in general, I would agree with you that yes, we always want to save to path when we are running the subsetting function for its purpose, i.e. subsetting the data for a data request. 

However, when I was writing the tests for the subsetting function, I realized that when we run the subsetting function in order to test its functionality, then we do NOT want it to save any data into the path. If we did that, then every time we run tests, the "testing subset" will be created anew (overwriting any other subset we may have done on this day). 

However, there may be better ways to fix that perhaps. Open to suggestions!

Reply:
Following a suggestion also from Stefan, I have now removed the `output_R` argument here.

Reply:
In the file `teardown-xyz.R` in the \test folder you can remove any files that were made during the test process. If you make the function save files in a different location (i.e. not overwriting any existing files) and then delete them after tests are done this would remove that issue.

Reply:
@LiamDBailey that's good to know regarding `teardown_xyz.R`. However, I do not really think that it can solve the problem here: if we want to save files to a different location during testing, then we still need some kind of `save`variable to specify the alternative save path...
I think I may just struggle a bit with thinking outside the box on this one. Could you check out the branch and implement your preferred solution? 
(I'll check back on it afterwards, so I can learn what works in this situation).